BlazarUI/app/index.mustache

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>BlazarUI</title>
+
+    <link rel="stylesheet" href="{{{ appRoot }}}{{{ staticRoot }}}css/vendor.css">
+    <link rel="stylesheet" href="{{{ appRoot }}}{{{ staticRoot }}}css/main.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/octicons/3.1.0/octicons.css">
```

Comment:
Yeah I had to add some of those urls from CDNs because right now we don't have any way to template the relative paths into the stylus files. I'll definitely look into a better solution.